src/Core/Framework/Bundle.php

```diff
@@ -100,20 +105,27 @@ public function configureRouteOverwrites(RouteCollectionBuilder $routes, string
         }
     }
 
-    /**
-     * @feature-deprecated tag:v6.4.0 - Implement `getActionEventClasses` instead
-     */
-    protected function getActionEvents(): array
+    public function getViewPaths(): array
     {
-        return [];
+        return [self::DEFAULT_PATH_VIEWS];
     }
 
-    /**
-     * Returns a list of all action event class references of this bundle. The events will be registered inside the `\Shopware\Core\Framework\Event\BusinessEventRegistry`.
```

Comment:
why did you remove the comments?

Reply:
I'm not sure why those were removed. I just them again